### ISSUE TITLE
wf_create_case_metadata catch Sumo time outs

### ISSUE THREAD

**Issue by perolavsvendsen** [equinor/fmu-dataio#281]
When user does not have access to Sumo, the `wf_create_case_metadata` will fail if the `--sumo` flag is used. From the user perspective, this looks like ERT is failing. A discussion is needed on how this is best handled.

1) Catch this and stop reporting error to ERT
2) At the same time, we must consider if there are users that _need_ this error to actually raise.

![image](https://user-images.githubusercontent.com/35033325/208638880-59797554-d051-41eb-add5-4f497079a379.png)

**Comment by perolavsvendsen** [comment]
Possibility is to add a new flag, e.g. `--require_sumo` or something which makes the script raise if error.

**Comment by slangeveld** [comment]
@perolavsvendsen Is this covered from the Sudo side?

**Comment by perolavsvendsen** [comment]
Yes, and it is also a remnant from early days when fmu-dataio was designed to be independent of Sumo.
